xml/QS_system_requirements.xml

```diff
@@ -17,7 +17,127 @@
  </para>
+ <sect1 xml:id="sec.caasp.quickstart.clusreqs">
+    <title>Cluster Size Requirements</title>
+    <para>
+     &productname; only functions in a clustered configuration. It requires a 
+     group of at least four physical or virtual machines. This will yield a 
+     cluster with one &admin_node;, one &master_node;, and two &worker_node;s.
+    </para>
+    <para>
+     All clusters must contain a single &admin_node; and at least one &master_node;
+     and one &worker_node;.
+    </para>
+    <para>
+     For improved performance and reliability, an odd number of &master_node;s 
+     and any number of &worker_node;s up may be added.
+    </para>
+    <note>
+     <title>Test and Proof-of-Concept Clusters</title>
+     <para>
+      It is possible to provision a three-node cluster with only a single
+      &worker_node;, but this is not a supported configuration for deployment.
+     </para>
+    </note>
+   </sect1>
+   <sect1 xml:id="sec.caasp.quickstart.hwreqs">
+    <title>Minimum Hardware Requirements</title>
+    <itemizedlist mark="bullet" spacing="normal">
+     <listitem>
+      <para>
+       Each node must have an AMD64/Intel* EM64T processor; quad-core
+       processors are recommended. 32-bit processors are not supported.
+      </para>
+     </listitem>
+     <listitem>
+      <para> 8 GB physical RAM per node. </para>
+     </listitem>
+     <listitem>
+      <para>
+       40 GB available disk space (or more depending on the count and size of 
+       the container images you intend to run).
+      </para>
+     </listitem>
+    </itemizedlist>
+   </sect1>
+   <sect1 xml:id="sec.caasp.quickstart.netreqs">
+    <title>Network Requirements</title>
+    <itemizedlist mark="bullet" spacing="normal">
+     <listitem>
+      <para>
+       In order to communicate without interruptions, all nodes should to be on
+       the same network.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       A DHCP server to dynamically provide IP addresses and host names for the
+       nodes in your cluster (unless you configure all nodes statically).
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       A DNS server to resolve host names. Please make sure you have reliable
+       DNS resolution at all times, especially in combination with DHCP.
+      </para>
+      <important>
+       <title>Unique Host Names</title>
+       <para>
+        All nodes' host names must be unique. It is recommended to let the DHCP
+        server provide not only the IP addresses but also the host names of the
+        cluster nodes. If two or more nodes have the same hostname, bootstrap of
+        the cluster will fail.
+       </para>
+      </important>
+     </listitem>
+     <listitem>
+      <para>
+       The &admin_node; must have a valid fully-qualified domain name (FQDN), 
+       which must be visible and accessible from a separate machine with a web
+       browser in order to complete the installation.
```

Comment:
...visible and accessible from a separate machine on your organization's network...

Is there any description in the following regarding the network requirements that will enable reaching the admin node?Gateway? Bastion  network on a second NIC?

Reply:
I have received some clarification of this elsewhere but I will need to get some more.